intro/setup/index.html

```diff
@@ -316,17 +316,54 @@ <h3>Exiting the game</h3>
 {% endhighlight %}
         
         And there we have it! Our game should close when we press the X button accordingly. 
+    </p>
+        
+    <p>
+        There are many different styles and ways to close a pygame application, 
+        you can raise an exception like shown here or use a variable to determine if the game is still running, 
+{% highlight python %}
+is_running = True 
```

Comment:
See below for a comment on this, TLDR: maybe name it `running` for simplicity while keeping readability